imgui/core.pyx

```diff
@@ -3273,6 +3293,475 @@ def drag_int4(
     ), (inout_values[0], inout_values[1], inout_values[2], inout_values[3])
 
 
+def input_text(str label, str value, cimgui.ImGuiInputTextFlags flags=0):
+    """Display text input widget.
+
+    .. visual-example::
+        :auto_layout:
+        :width: 200
+        :height: 100
+
+        text_val = 'Please, type the coefficient here.'
+        imgui.begin("Example: text input")
+        changed, text_val = imgui.input_text('Amount:', text_val)
```

Comment:
Label will be displayed on right so semicolon looks weird in the example.
You can either use hidden label with `##` prefix:
```python
changed, text_val = imgui.input_text('##Amount:', text_val)
```
or simply use `imgui.same_line()`.